tests/system/action/user/test_assign_meetings.py

```diff
@@ -0,0 +1,242 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class UserAssignMeetings(BaseActionTestCase):
+    def test_assign_meetings_correct(self) -> None:
+        self.set_models(
+            {
+                "group/11": {"name": "to_find", "meeting_id": 1, "user_ids": [1]},
+                "group/22": {"name": "nothing", "meeting_id": 2, "user_ids": [1]},
+                "group/31": {"name": "to_find", "meeting_id": 3},
+                "group/43": {"name": "standard", "meeting_id": 4},
+                "group/51": {"name": "to_find", "meeting_id": 5},
+                "group/52": {"name": "nothing", "meeting_id": 5, "user_ids": [1]},
+                "meeting/1": {
+                    "name": "success(existing)",
+                    "group_ids": [11],
+                    "is_active_in_organization_id": 1,
+                    "committee_id": 2,
+                },
+                "meeting/2": {
+                    "name": "nothing",
+                    "group_ids": [22],
+                    "is_active_in_organization_id": 1,
+                    "committee_id": 2,
+                },
+                "meeting/3": {
+                    "name": "success(added)",
+                    "group_ids": [31],
+                    "is_active_in_organization_id": 1,
+                    "committee_id": 2,
+                },
+                "meeting/4": {
+                    "name": "standard",
+                    "group_ids": [43],
+                    "is_active_in_organization_id": 1,
+                    "default_group_id": 43,
+                    "committee_id": 2,
+                },
+                "meeting/5": {
+                    "name": "standard",
```

Comment:
With your "naming standard" this should be `success(added)`

Reply:
Done.